### PR TITLE
fix config example

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

**Review by anandjakhaniya — APPROVED** [review]

File: backend/config.example.yaml [hunk 1]
```diff
@@ -8,16 +8,15 @@ springServer:
...
   devKey:
 
 artifactStoreConfig:
   artifactStoreType: NFS #S3, GCP, NFS
   pickArtifactStoreHostFromConfig: false
-  artifactStoreServerHost: localhost:8086
-  artifactStoreUrlProtocol: http
-  artifactStoreServerPort: 8086
+  host: localhost:8086
+  protocol: http
```
Comment (anandjakhaniya) [inline backend/config.example.yaml:21]: there is one missing field in config here : `enabled`,

Comment (anandjakhaniya) [reply backend/config.example.yaml:21]: there are many other fields are missing in config file like in s3 config `minioEndpoint:` is missing etc. need to verify entire config file.

**Review by anandjakhaniya — CHANGES_REQUESTED** [review]
Needs to be check whole config file for missing parameters

**Comment by conradoverta** [comment]
Is this still relevant?

**Comment by evgeniilezhnin** [comment]
> Is this still relevant?

yes, but I forgot to add other missing fields, sorry

**Review by anandjakhaniya — APPROVED** [review]

**Comment by evgeniilezhnin** [comment]
Anands changes LGTM

**Pull request merged**